src/sysupdate/sysupdated.c

```diff
@@ -75,6 +76,7 @@ typedef enum JobType {
         JOB_CHECK_NEW,
         JOB_UPDATE,
         JOB_VACUUM,
+        JOB_DESCRIBE_FEATURE,
```

Comment:
Make sure I'm looking for all switch cases where these are used, and adding this case.

For example: job_method_cancel will panic the daemon if you try to cancel on of these jobs.